src/AtmosphereModels/update_atmosphere_model_state.jl

```diff
@@ -154,7 +159,7 @@ end
 
     update_microphysical_fields!(microphysical_fields, microphysics,
                                  i, j, k, grid,
-                                 ρ, 𝒰₁, constants)
+                                 ρ, 𝒰₁, p′, constants, Δt)
```

Comment:
can you explain why the perturbation pressure is needed and what it represents? Can it be computed from the density?

More generally we want to collapse the thermodynamic state into as few objects as possible, so I am wondering if this warrants a better design.

Also in what sens is this pressure a "perturbation"?